README.md

```diff
@@ -1,75 +1,72 @@
 
-> Description the project.
+## Built with
 
+- HTML CSS & JAVASCRIPT ES6
 
-## Built With
+## Additional Tools
 
-- Major languages
-- Frameworks
-- Technologies used
+- WebHint
+- Stylelint
+- ESlint
 
-## Live Demo (if available)
+## Live demo
 
-[Live Demo Link](https://livedemo.com)
+You can watch [here](/) a live demo of our project.
 
+## Test
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+To track linter errors locally follow these steps:
 
-### Prerequisites
+Download all the dependencies run:
 
-### Setup
+```
+npm install
+```
 
-### Install
+Track HTML linter errors run:
 
-### Usage
+```
+npx hint .
+```
 
-### Run tests
+Track CSS linter errors run:
 
-### Deployment
+```
+npx stylelint "**/*.{css,scss}"
+```
 
+Track JavaScript linter errors run:
 
+```
+npx eslint ."
+```
 
 ## Authors
 
-👤 **Author1**
+👤 **Petro Loltolo Lesapiti**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@petrolesapiti](https://github.com/Loltolo-Lesapiti)
+- LinkedIn: [@petrolesapiti](https://www.linkedin.com/in/petrolesapitiloltolo/)
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## Collaborators
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
+## Getting Started
 
-Give a ⭐️ if you like this project!
+To get a local copy follow these simple steps:
 
-## Acknowledgments
+- Copy this link .
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Create a local directory that you want to clone the repository.
 
-## 📝 License
+- Open the command prompt in the created directory.
 
-This project is [MIT](./LICENSE) licensed.
+- Go to the repository folder using command prompt cd javascript-capstone.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+- Install the dev dependencies for linters run npm to install.
```

Comment:
- [ ] I would also recommend adding the `Acknowledgments` and the `Contributing` sections with the issue link in : 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).


## Acknowledgments

- Microverse
```



- Don't forget to add the ``` ##  License ``` section  at the end of your README file to make it more professional. I would also recommend you add this. 😄 


- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.